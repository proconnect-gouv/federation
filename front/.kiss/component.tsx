import React from 'react';

interface DumbComponentProps {
  className: string;
}

export const DumbComponent: React.FC<DumbComponentProps> = React.memo(
  ({ className }: DumbComponentProps) => {
    return <div />;
  },
);

DumbComponent.displayName = 'DumbComponent';
