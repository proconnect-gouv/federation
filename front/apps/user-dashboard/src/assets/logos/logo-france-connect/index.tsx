import LogoSVG from './logo.svg';

type LogoFranceConnectPlusProps = {
  className?: string;
};

const LogoFranceConnectComponent = ({
  className,
}: LogoFranceConnectPlusProps) => (
  <div className={className}>
    <img
      alt="marianne hexagonale franceconnect plus"
      height="auto"
      src={LogoSVG}
      width="140"
    />
  </div>
);

LogoFranceConnectComponent.defaultProps = {
  className: '',
};

LogoFranceConnectComponent.displayName = 'LogoFranceConnectComponent';

export default LogoFranceConnectComponent;
