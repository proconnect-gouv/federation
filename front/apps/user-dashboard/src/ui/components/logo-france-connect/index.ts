/* istanbul ignore file */

// declarative file
export * from './logo-france-connect.component';
