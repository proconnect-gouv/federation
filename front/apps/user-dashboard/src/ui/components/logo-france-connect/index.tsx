import LogoSVG from './logo.svg';

type LogoFranceConnectPlusProps = {
  className?: string;
};

export const LogoFranceConnectComponent = ({
  className,
}: LogoFranceConnectPlusProps) => (
  <div className={className}>
    <img alt="marianne hexagonale franceconnect plus" src={LogoSVG} />
  </div>
);

LogoFranceConnectComponent.defaultProps = {
  className: '',
};

LogoFranceConnectComponent.displayName = 'LogoFranceConnectComponent';
