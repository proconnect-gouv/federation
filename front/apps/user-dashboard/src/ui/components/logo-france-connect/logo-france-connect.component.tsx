import LogoSVG from './logo.svg';

interface LogoFranceConnectPlusProps {
  className?: string;
}

export const LogoFranceConnectComponent = ({ className }: LogoFranceConnectPlusProps) => (
  <div className={className}>
    <img alt="marianne hexagonale franceconnect plus" src={LogoSVG} />
  </div>
);

LogoFranceConnectComponent.defaultProps = {
  className: '',
};

LogoFranceConnectComponent.displayName = 'LogoFranceConnectComponent';
