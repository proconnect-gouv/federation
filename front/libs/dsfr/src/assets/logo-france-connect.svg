<svg xmlns="http://www.w3.org/2000/svg" width="144" height="68"><defs><linearGradient id="a" x1=".027" x2="63.764" y1="31.87" y2="31.87" gradientTransform="scale(.93709 1.06713)" gradientUnits="userSpaceOnUse"><stop offset="0%" stop-color="#000091"/><stop offset="48.88%" stop-color="#844470"/><stop offset="100%" stop-color="#E1000F"/></linearGradient></defs><g style="fill:none;fill-rule:evenodd"><g style="fill-rule:nonzero"><path d="M30.504 1.173c-.316-.188-.885-.188-1.201 0L1.62 17.104c-.316.19-.568.63-.568 1.008v31.799c0 .378.252.819.568 1.007L29.303 66.85c.316.19.885.19 1.2 0l27.683-15.93c.316-.19.569-.63.569-1.008v-31.8c0-.378-.253-.819-.569-1.008L30.504 1.174Z" style="fill:#ffffff"/><path d="m30.44 1.173 27.683 15.931c.316.19.569.63.569 1.008v31.799c0 .378-.253.819-.569 1.007L30.44 66.85c-.19.063-.379.126-.568.126a1.52 1.52 0 0 1-.57-.126L1.62 50.92c-.316-.19-.568-.63-.568-1.008v-31.8c0-.378.252-.819.568-1.008l27.683-15.93c.126-.064.379-.126.569-.126.19 0 .442.062.568.125zM28.797.292 1.115 16.222C.483 16.602.04 17.42.04 18.113v31.799c0 .756.443 1.511 1.075 1.889l27.682 15.931c.38.252.822.252 1.075.252.252 0 .695-.063 1.074-.252L58.63 51.8c.632-.378 1.074-1.196 1.074-1.889V18.112c0-.756-.442-1.511-1.074-1.89L31.009.293C30.567.04 30.188.04 29.872.04c-.253 0-.696.063-1.075.252z" style="fill:url(#a)"/><path d="m21.78 17.235-.065-.065c.064 0 .064 0 .064.065z" style="fill:#ea212e"/><path d="M29.799 59.723 8.23 47.283c-.252-.126-.441-.505-.441-.758v-24.5c0-.316.19-.632.441-.758L29.86 8.764" style="fill:#0053cc"/><path d="M7.853 46.415 7.79 21.881c0-.317.19-.633.441-.76l21.63-12.52" style="fill:#000091"/><path d="m8.277 21.017 6.635 13.389 7.646-13.264-6.508-4.588Z" style="fill:#e1000f"/><path d="m37.944 17.235.065-.065c-.065 0-.065 0-.065.065z" style="fill:#ea212e"/><path d="m29.925 59.56 21.567-12.424c.252-.126.441-.504.441-.756V21.909c0-.315-.189-.63-.441-.757l-21.63-12.55" style="fill:#000091"/><path d="m51.87 46.415.063-24.516c0-.315-.189-.63-.441-.756L29.862 8.601" style="fill:#e1000f"/><path d="m51.284 20.949-6.509 13.294-7.773-13.357 6.446-4.495Z" style="fill:#0053cc"/></g><path d="M38.06 20.38c.507.255.979.583 1.306 1.093.145.219.326.62.217.912-.145.327-.217.91-.544 1.02-.4.22-.944.22-1.452.146-.29 0-.544-.073-.834-.146 1.016.401 1.996.911 2.685 1.86.073.145.327.218.617.218.073 0 .073.146.073.219-.145.146-.29.219-.218.4h.218c.326-.145.29-.837.762-.619.326.219.471.693.29 1.021-.29.292-.544.474-.835.693a.547.547 0 0 0 0 .474c.218.292.29.547.327.838.218.474.29 1.021.472 1.531.29 1.021.544 2.078.471 3.1 0 .546-.29 1.02-.072 1.567.145.547.471.948.762 1.458.29.401.544.693.762 1.094.088.154.195.308.304.463l.109.155c.342.494.651 1.005.422 1.57-.218.473-.944.4-1.452.692-.399.328-.072.912.145 1.24.327.62-.399 1.02-.907 1.24.145.218.4.145.471.29.073.329.4.548.219.912a1.735 1.735 0 0 1-.334.313l-.095.074c-.28.222-.518.454-.26.853.29.474.108 1.02-.074 1.531-.218.62-.762.911-1.233 1.021-.4.145-.908.145-1.306.073-.146-.073-.291-.146-.4-.146-1.161-.146-2.322-.474-3.483-.474-.327.073-.69.146-.944.255-.327.219-.58.474-.835.73-.036.072-.109.108-.145.182-.036.036-.072.072-.072.109l-.022.022-.03.03-.02.02c-.219.256-.364.511-.545.803 0 .036-.037.036-.037.036 0 .037-.036.073-.072.11-.218.4-.4.838-.508 1.276-.472 1.567-.254 2.916.072 3.245.218.218 2.831 1.028 5.13 1.782-4.016 3.354-7.262 5.03-9.738 5.03-3.419 0-9.153-2.271-17.203-6.815 1.949 0 1.671.78 3.486-.128.87-.364 2.83-1.273 3.484-1.927.399-.4.798-.8 1.16-1.236.727-.836 1.452-1.6 2.287-2.327.254-.218.508-.437.762-.582.073-.073.073-.218.145-.29-.326.145-.544.4-.907.545-.073 0-.145-.073-.073-.146.254-.182.508-.363.726-.545h-.036c-.073 0-.073-.073-.073-.146-.907-.145-1.56.473-2.177 1.018-.145.073-.29-.072-.326-.072-1.017.327-1.779 1.236-2.795 1.636v-.146c-.399.146-.798.4-1.233.473-.617.145-1.162.073-1.706.073-.835.072-1.67.254-2.504.436-.036 0-.036 0-.072.036-.436.11-.871.291-1.27.51l-.037.036-.109.109c-.145.145-.29.327-.472.4-.435.218-.762.581-1.125.909-.036.036-.072.036-.108.036-.363.364.072-.727.471-.982-.036 0-.109-.036-.072-.072.036-.073.072-.11.109-.182 0-.037 0-.037.036-.073l-.006-.022-.024-.03a.03.03 0 0 1-.006-.02c-.11.072-.218.145-.327.218-.181.145-.109-.691.073-.8h-.037c.254-.146.545-.291.799-.437l.108-.109c-.399.146-.725.328-1.088.546 0 0 0-.255.11-.364.071 0 .144 0 .144.073 1.316-1.016 2.914-1.287 4.528-1.522l.44-.064c.954-.14 1.905-.29 2.798-.596.217-.145.399-.29.616-.4.327-.145.617-.472 1.016-.69.545-.4.944-.91 1.162-1.563 0-.073-.072-.146-.072-.146-.908.945-1.924 1.709-3.013 2.254-1.451.764-3.012.618-4.536.837.073-.146.218-.146.327-.146 0-.218.145-.291.29-.4h.218c.072 0 .072-.145.145-.145.145 0 .363-.073.29-.073-.217-.291-.616.218-.943 0 .145-.145.073-.327.218-.4h.29c0-.146.145-.29.145-.29 1.016-.62 1.996-1.091 2.94-1.637-.218 0-.327.218-.545.073.146 0 0-.218.146-.218.761-.218 1.378-.618 2.14-.91-.29 0-.472.219-.762 0 .145-.072.218-.217.4-.217v-.218c0-.074.072-.074.144-.074-.072 0-.145-.072-.145-.072.073-.146.291-.073.4-.218-.073 0-.218 0-.218-.073.217-.29.544-.327.907-.4-.073-.145-.29 0-.29-.145 0-.063.053-.071.114-.073h-.114c-.146-.073-.073-.218-.073-.291.4-.472.4-1.09.617-1.636-.073 0-.145 0-.145-.072-.69.763-1.778 1.018-2.794 1.308h-.472c-.326.146-.835.146-1.161-.072-.29-.146-.4-.328-.69-.546-.544-.327-1.088-.618-1.705-.836-1.706-.545-3.484-.836-5.262-.763.762-.4 1.597-.437 2.395-.691 1.161-.327 2.25-.764 3.484-.691-.218-.073-.472 0-.69 0-.944-.073-1.923.218-2.94.4-.689.145-1.306.4-1.995.545-.4.146-.617.546-1.089.473v-.218c.69-.836 1.525-1.636 2.613-1.709 1.234-.218 2.395 0 3.629.146.907.072 1.705.29 2.612.472.327 0 .4.545.69.618.4.145.835 0 1.234.29 0-.145-.073-.29 0-.399.29-.29.617.073.907-.073.544-.327-.472-.945-.762-1.454 0-.073.073-.145.073-.145.544.472.943 1.018 1.633 1.381.326.146 1.16.327 1.015-.072-.326-.764-1.015-1.382-1.596-2.073v-.29c-.145 0-.145-.073-.218-.146v-.29c-.29-.146-.218-.401-.326-.62-.218-.326-.073-.835-.218-1.235a6.784 6.784 0 0 1-.29-1.164c-.218-1.163-.472-2.181-.617-3.308-.145-1.309.762-2.327 1.379-3.49.471-.836 1.016-1.636 1.923-2.182.218-.836.762-1.527 1.306-2.18.545-.656 1.452-1.091 2.105-1.383.876-.404 1.34-.547 1.802-.686.002-.001.01-.003.012-.005.67-.294 9.477-2.146 13.753 0 .091.046.073 0 .363.183.327.182.726.4.98.51zM4.833 40.32c0 .32-.215.604-.536.675.536.142 1.073.142 1.61.391-.072.142-.215.071-.286.071.322.213.751.071 1.074.32-.215.213-.394 0-.609 0 2.112.604 4.33 1.066 6.085 2.417-1.503.747-3.042 1.067-4.653 1.422-.215 0-.322 0-.537-.071 0 .071 0 .213-.071.213-.287 0-.466 0-.68.142-.252.214-.645.285-.86.071-.071-.07.215 0 .287-.141H5.12c-.072 0-.072-.072-.072-.142-.322.07-.751.212-1.074.283-.465.143-.25-2.168.072-2.31-.072-.071-.215-.071-.215-.213-.286 0-.393-.711-.215-.747.287-.07.609-.07.824-.284-.466-.071-.609-1.742.178-2.169.072 0 .215 0 .215.072zm3.118 6.279c-.434.445-.977.243-1.446.445-.146 0-.399.081-.326-.08.045-.229.246-.314.46-.385l.13-.043c.086-.028.17-.058.242-.1.144-.08.325-.242.47-.161.144.243.325.162.47.324zM5.49 37.286c-.072 0-.146 0-.219-.073.073-.073.073-.147.073-.22-.694.768-2.264.512-.803-.183.548-.22 1.241-.146 1.79.073-.22.293-.549.22-.84.403zm33.42-4.57a3.182 3.182 0 0 1-.19.037c-.332 0-.702.215-1.035.287-.48.143-.924.394-1.33.681.48-.215.96-.287 1.515-.394.407-.072.74-.144 1.183-.072.296.072.703.072.703.215-.148.538-.962.68-1.405 1.218h-.222c-.222.144-.148.466-.333.466-.222-.071-.407 0-.629.072.296.287.63.466 1.036.394.074 0 .222.143.222.286 0 0 .074 0 .148-.071.074 0 .147 0 .147.071v.287c-.132.172-.306.19-.487.191h-.121a1.092 1.092 0 0 0-.242.024c.555.143 1.11.143 1.627 0 .407-.143 0-.824.296-1.146-.148 0 0-.215-.148-.215.148-.144.296-.323.407-.395.148 0 .333-.071.407-.215 0-.143-.296-.214-.222-.322.406-.286.776-.68.628-1.075-.074-.215-.628-.215-.961-.358-.23-.1-.514-.061-.802-.005l-.192.04z" style="fill:#ffffff"/><path style="fill:#19199c;fill-rule:nonzero" d="M69.897 32.782v-4.803h3.668v-1.964h-3.668v-2.63h4.317v-1.963h-6.621v11.36zm8.131 0v-5.128c.152-.249.392-.487.722-.714.33-.227.744-.34 1.242-.34.184 0 .349.01.495.032.146.021.284.048.414.08v-2.157a2.24 2.24 0 0 0-.715-.113c-.465 0-.873.094-1.225.284a3.976 3.976 0 0 0-.933.69v-.814h-2.061v8.18zm6.865.325c.476 0 .936-.1 1.38-.3.443-.2.795-.463 1.054-.787v.762h2.078v-5.29c0-.444-.076-.863-.228-1.258a2.79 2.79 0 0 0-.673-1.023 3.215 3.215 0 0 0-1.128-.681c-.454-.168-.974-.252-1.558-.252-.736 0-1.398.146-1.988.439a3.912 3.912 0 0 0-1.436 1.184l1.525 1.153a1.87 1.87 0 0 1 .755-.723c.32-.167.679-.251 1.08-.251.464 0 .843.127 1.135.381.292.254.438.598.438 1.03v.212l-2.466.422c-.877.14-1.55.438-2.021.892-.47.455-.706.99-.706 1.607 0 .768.257 1.374.77 1.818.515.443 1.178.665 1.989.665zm.422-1.655c-.346 0-.622-.087-.828-.26a.817.817 0 0 1-.308-.65c0-.259.095-.475.284-.648.19-.174.5-.298.933-.374l1.931-.34v1.12c-.205.335-.476.61-.811.827-.336.216-.736.325-1.201.325zm8.504 1.33v-5.339a3.58 3.58 0 0 1 .674-.803c.297-.276.705-.414 1.225-.414.497 0 .89.16 1.176.479.287.319.43.749.43 1.29v4.787h2.095v-4.868c0-.606-.084-1.136-.252-1.59a3.263 3.263 0 0 0-.681-1.137 2.763 2.763 0 0 0-1.006-.681 3.344 3.344 0 0 0-1.242-.228c-.54 0-1.009.095-1.404.284a3.87 3.87 0 0 0-1.014.69v-.65h-2.061v8.18zm11.863.325c.747 0 1.415-.152 2.005-.454a4.21 4.21 0 0 0 1.452-1.201l-1.639-1.25a2.32 2.32 0 0 1-.755.69 2.085 2.085 0 0 1-1.063.267c-.681 0-1.246-.232-1.695-.697-.45-.466-.674-1.055-.674-1.77 0-.713.222-1.303.665-1.768.444-.466 1.001-.698 1.672-.698.422 0 .787.09 1.095.268.309.178.56.408.755.69l1.64-1.25a4.231 4.231 0 0 0-1.453-1.201c-.59-.303-1.258-.455-2.005-.455-.703 0-1.336.12-1.898.357a4.214 4.214 0 0 0-1.42.958c-.384.4-.68.868-.885 1.404a4.698 4.698 0 0 0-.308 1.696c0 .595.103 1.16.308 1.696.206.535.5 1.003.885 1.403.384.4.857.72 1.42.958.562.238 1.195.357 1.898.357zm8.959 0c.768 0 1.46-.149 2.077-.446a3.963 3.963 0 0 0 1.493-1.21l-1.476-1.168a2.372 2.372 0 0 1-.86.706 2.648 2.648 0 0 1-1.202.268c-.692 0-1.26-.19-1.704-.568-.443-.379-.697-.904-.762-1.574h5.728a5.421 5.421 0 0 0 .13-1.169c0-1.071-.346-1.95-1.038-2.637-.693-.687-1.602-1.03-2.727-1.03-.67 0-1.266.116-1.785.348-.52.233-.958.55-1.315.95-.357.4-.63.868-.82 1.403a5.114 5.114 0 0 0-.283 1.713c0 .595.1 1.16.3 1.696.2.535.492 1.003.876 1.403.385.4.86.72 1.429.958.568.238 1.214.357 1.939.357zm1.38-5.55h-3.685c.141-.53.38-.923.715-1.177.335-.254.751-.381 1.25-.381.54 0 .96.16 1.257.478.297.32.452.68.462 1.08z"/><path style="fill:#ed1c24;fill-rule:nonzero" d="M72.656 47.389c1.028 0 1.953-.211 2.775-.633a5.687 5.687 0 0 0 2.013-1.688l-1.818-1.396a3.766 3.766 0 0 1-1.241 1.128c-.503.287-1.08.43-1.729.43-.53 0-1.022-.1-1.477-.3-.454-.2-.84-.47-1.16-.812-.32-.34-.57-.746-.755-1.217a4.126 4.126 0 0 1-.276-1.517c0-.541.092-1.047.276-1.518.184-.47.436-.876.755-1.217.32-.34.706-.611 1.16-.811.455-.2.947-.3 1.477-.3.65 0 1.226.143 1.729.43.503.286.917.662 1.241 1.128l1.818-1.412a5.907 5.907 0 0 0-2.013-1.664c-.822-.427-1.747-.641-2.775-.641a6.069 6.069 0 0 0-4.365 1.785 5.746 5.746 0 0 0-1.234 1.908c-.292.73-.438 1.5-.438 2.312 0 .811.146 1.582.438 2.312.292.73.703 1.366 1.234 1.907.53.541 1.168.974 1.915 1.299a6.086 6.086 0 0 0 2.45.487zm10.403 0c.66 0 1.266-.117 1.818-.35a4.4 4.4 0 0 0 1.42-.94c.395-.396.7-.863.917-1.404a4.598 4.598 0 0 0 .324-1.72c0-.607-.108-1.18-.324-1.721a4.138 4.138 0 0 0-.917-1.404 4.386 4.386 0 0 0-1.42-.941 4.628 4.628 0 0 0-1.818-.35c-.66 0-1.266.117-1.818.35a4.397 4.397 0 0 0-1.42.941c-.395.395-.7.863-.917 1.404a4.594 4.594 0 0 0-.324 1.72c0 .606.108 1.18.324 1.72.217.542.522 1.01.917 1.404.395.395.869.709 1.42.942a4.646 4.646 0 0 0 1.818.349zm.033-1.948c-.682 0-1.247-.233-1.696-.698-.45-.465-.674-1.055-.674-1.769s.225-1.303.674-1.769c.449-.465 1.014-.698 1.696-.698.649 0 1.195.233 1.639.698.443.466.665 1.055.665 1.77 0 .703-.222 1.29-.665 1.76-.444.47-.99.706-1.64.706zm8.39 1.623v-5.34c.151-.259.376-.527.674-.803.297-.276.705-.414 1.225-.414.498 0 .89.16 1.176.48.287.318.43.748.43 1.29v4.787h2.094v-4.869c0-.606-.084-1.136-.251-1.59a3.275 3.275 0 0 0-.682-1.136 2.778 2.778 0 0 0-1.006-.682 3.364 3.364 0 0 0-1.242-.227c-.54 0-1.009.095-1.404.284a3.87 3.87 0 0 0-1.014.69v-.65h-2.061v8.18zm9.997 0v-5.34c.152-.259.376-.527.674-.803.297-.276.706-.414 1.225-.414.498 0 .89.16 1.177.48.286.318.43.748.43 1.29v4.787h2.093v-4.869c0-.606-.084-1.136-.251-1.59a3.275 3.275 0 0 0-.682-1.136 2.778 2.778 0 0 0-1.006-.682 3.364 3.364 0 0 0-1.242-.227c-.54 0-1.009.095-1.404.284-.394.19-.733.42-1.014.69v-.65h-2.06v8.18h2.061zm11.896.325c.768 0 1.46-.15 2.077-.447a3.961 3.961 0 0 0 1.493-1.209l-1.476-1.168a2.374 2.374 0 0 1-.86.706 2.66 2.66 0 0 1-1.201.267c-.693 0-1.26-.189-1.705-.568-.443-.378-.697-.903-.762-1.574h5.729a5.73 5.73 0 0 0 .089-.52c.027-.205.04-.421.04-.648 0-1.071-.346-1.95-1.038-2.638-.693-.687-1.602-1.03-2.727-1.03-.67 0-1.266.116-1.785.349-.52.232-.957.549-1.314.95-.358.4-.63.868-.82 1.403a5.083 5.083 0 0 0-.284 1.712c0 .595.1 1.16.3 1.696.2.536.492 1.004.877 1.404.384.4.86.72 1.428.958s1.214.357 1.94.357zm1.38-5.55h-3.684c.14-.53.378-.923.714-1.177.335-.255.752-.382 1.25-.382.54 0 .96.16 1.257.479.298.32.452.679.463 1.08zm7.984 5.55c.747 0 1.415-.152 2.005-.455a4.218 4.218 0 0 0 1.452-1.2l-1.64-1.25c-.194.28-.445.51-.754.69a2.085 2.085 0 0 1-1.063.267c-.681 0-1.247-.233-1.696-.698-.449-.465-.673-1.055-.673-1.769s.222-1.303.665-1.769c.444-.465 1.001-.698 1.672-.698.422 0 .787.09 1.095.268.309.179.56.409.755.69l1.64-1.25a4.229 4.229 0 0 0-1.453-1.2c-.59-.304-1.258-.455-2.005-.455-.703 0-1.336.119-1.899.357a4.22 4.22 0 0 0-1.42.958 4.16 4.16 0 0 0-.884 1.403 4.685 4.685 0 0 0-.308 1.696c0 .595.102 1.16.308 1.696.206.536.5 1.004.884 1.404.385.4.858.72 1.42.958.563.238 1.196.357 1.9.357zm8.537-.163c.335 0 .63-.021.884-.065.255-.043.484-.108.69-.194v-1.802a2.56 2.56 0 0 1-.503.13 4.488 4.488 0 0 1-.714.049c-.4 0-.717-.098-.95-.292-.232-.195-.348-.52-.348-.974v-3.31h2.499v-1.883h-2.5V36.84h-2.077v2.045h-1.525v1.882h1.525v3.31c0 .996.249 1.77.747 2.322.497.551 1.255.827 2.272.827z"/></g></svg>
