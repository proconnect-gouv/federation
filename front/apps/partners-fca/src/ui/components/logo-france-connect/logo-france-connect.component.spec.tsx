import { render } from '@testing-library/react';

import { LogoFranceConnectComponent } from './logo-france-connect.component';

describe('LogoFranceConnectComponent', () => {
  it('should match the snapshot', () => {
    const { container } = render(<LogoFranceConnectComponent />);

    expect(container.firstChild).toMatchSnapshot();
  });

  it('should render an element with a given classname', () => {
    // given
    const { container } = render(<LogoFranceConnectComponent className="mock-classname" />);
    // then
    const elements = container.getElementsByClassName('mock-classname');
    expect(elements).toHaveLength(1);
  });
});
