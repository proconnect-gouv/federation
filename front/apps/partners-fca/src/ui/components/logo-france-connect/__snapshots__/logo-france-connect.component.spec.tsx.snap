// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogoFranceConnectComponent should match the snapshot 1`] = `
<div>
  <img
    alt="marianne hexagonale franceconnect plus"
    src="SvgrURL"
  />
</div>
`;
